src/lib/radio/radio.spec.ts

```diff
@@ -760,3 +784,23 @@ class RadioGroupWithFormControl {
   template: `<mat-radio-button tabindex="-1"></mat-radio-button>`
 })
 class FocusableRadioButton {}
+
+@Component({
+  template: `
+  <mat-list>
```

Comment:
In general we try to avoid mixing Material components between from different modules each other. You should be able to test the same either by wrapping the radio button in a div, or adding a custom component that is only used in the tests.

Reply:
OK, that makes sense. I will update the tests accordingly.